fastcrypto-vdf/src/class_group/reduction.rs

```diff
@@ -54,3 +54,34 @@ impl QuadraticForm {
         }
     }
 }
+
+#[cfg(test)]
```

Comment:
after this is merged I'd expect some fuzzy test coverage here (even via LLM aid)